dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -908,8 +920,20 @@ export const AdSlot = ({
 	}
 };
 
-export const MobileStickyContainer = () => {
+type MobileStickyContainerProps = Pick<FEArticleType, 'contentType' | 'pageId'>;
+
+export const MobileStickyContainer = ({
+	contentType,
+	pageId,
+}: MobileStickyContainerProps) => {
 	return (
-		<div className="mobilesticky-container" css={mobileStickyAdStyles} />
+		<div
+			className="mobilesticky-container"
+			css={[
+				mobileStickyAdStyles,
+				(contentType === 'Article' || pageId.startsWith('football/')) &&
+					mobileStickyAdStylesFullWidth,
```

Comment:
I'm maybe missing the context here, but why do we want to apply the full width styles only to articles and football pages?

Reply:
I got the logic behind it from Commercial https://github.com/guardian/commercial/blob/5151be8140b45d4ceebbb9629f27ffe1b41147ef/src/lib/header-bidding/utils.ts#L43-L46

I just had to find a way to override the styling so there has to be a condition.

Reply:
I wonder if we should export that logic via commercial core, so we only need to update it in one place? I notice [`shouldIncludeMobileSticky`](https://github.com/guardian/commercial/blob/5151be8140b45d4ceebbb9629f27ffe1b41147ef/src/lib/header-bidding/utils.ts#L197-L207) is exported and used in [a couple of places](https://github.com/search?q=repo%3Aguardian%2Fcommercial%20shouldIncludeMobileSticky&type=code) around the commercial codebase, so might be a better target for reuse.

Reply:
When you say commercial core you mean "commercial-shared" repository?